index.js

```diff
@@ -44,6 +44,10 @@ module.exports = function generateRequireForUserCode (scopedDirs, options) {
   })
 
   function deleteModuleFromCache (m) {
+    if (m.id.endsWith('.node')) {
```

Comment:
Let's be on the safe side and make sure that the properties chain is defined.
m && m.id && m.id.endsWith

Reply:
will do, but I guess if its not the case wouldn't the next instuctions fail?